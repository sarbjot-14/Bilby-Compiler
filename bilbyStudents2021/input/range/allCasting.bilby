main{
	
    mut r := [<44 .. 77> as <int>];
    print r $n;
    
    mut a:= [<44.4 .. 77.7> as <float>];
    print a $n;
    
    %mut n := [<#d .. #i> as <char>];
    %print n $n;
    

	
}