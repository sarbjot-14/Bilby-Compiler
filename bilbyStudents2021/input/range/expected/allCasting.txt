<,..MNot enough operands on stack for Printf instruction at location 67.
